en/docs/develop/deploying-the-sample-app.md

```diff
@@ -4,87 +4,364 @@ This topic provides instructions on how to download and deploy the
+    
+    | Properties                                                                                                                                                                          | Description                                                        |
+    |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------|
+    | `             SAML2.SPEntityId=travelocity.com                         `                                                                                                            | A unique identifier for this SAML 2.0 Service Provider application |
+    | `              SAML2.AssertionConsumerURL=                                             http://wso2is.local:8080/travelocity.com/home.jsp                                          ` | The URL of the SAML 2.0 Assertion Consumer                         |
+    | `              SAML2.IdPURL=                                             https://localhost:9443/samlsso                                          `                                  | The URL of the SAML 2.0 Identity Provider                          |
+    | `             SAML2.IsPassiveAuthn=true                         `                                                                                                                   | Set this to send SAML2 passive authentication requests             |
+    
+    If you edit the **travelocity.properties** file, you must restart the
+    Apache Tomcat server for the changes to take effect.
+    
+
+    
+### Configuring Service Provider
+
+The next step is to configure the service provider.
+
+1.  Return to the WSO2 IS management console.
+
+2.  Navigate to **Main** tab -> **Identity** -> **Service Providers** -> **Add**.
+
+3.  Enter **travelocity.com** in the **Service Provider Name** text box,
+    and click **Register**.
+
+4.  In the **Inbound Authentication Configuration** section, click
+    **Configure** under the **SAML2 Web SSO Configuration** section.
+
+    1.  Now set the configurations as follows:
+
+        1.  **Issuer** : `               travelocity.com              `
+
+        2.  **Assertion Consumer URL** :
+            `                               http://wso2is.local:8080/travelocity.com/home.jsp                        `  
+            Click Yes, in the message that appears.
+
+    2.  Select the following check-boxes:
+        1.  **Enable Response Signing**
+
+        2.  **Enable Single Logout**
+
+        3.  **Enable Attribute Profile**
+
+        4.  **Include Attributes in the Response Always**  
+        
+        5.  **Enable Signature Validation in Authentication Requests and Logout Requests**
+            
+
+    ![edit-service-provider](../assets/img/tutorials/edit-service-provider-configs.png)
```

Comment:
This image in not rendered properly in https://is.docs.wso2.com/en/5.9.0/develop/deploying-the-sample-app/#deploying-travelocity-webapp